src/components/InlineTooltip.js

```diff
@@ -29,7 +31,9 @@ var InlineTooltip = React.createClass({
         style: React.PropTypes.object,
         icon: React.PropTypes.string,
         iconStyle: React.PropTypes.object,
-        timeout: React.PropTypes.number
+        timeout: React.PropTypes.number,
+        node : React.PropTypes.node,
```

Comment:
Do you think "node" describes it the best? I'm not pushing, but may be it's better to use something like "title" or "caption"?